src/pages/IndexPage.vue

```
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import { useMessagesStore } from "@/stores/messages";
+import MessageItem from "~/layout/MessageItem.vue";
```

ESLint

[error] 'MessageItem' is defined but never used.

(@typescript-eslint/no-unused-vars)